x/migration/keeper/msg_server_claim_morse_application.go

```diff
@@ -14,6 +13,14 @@ import (
 	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
+// ClaimMorseApplication performs the following steps, given msg is valid and a
+// MorseClaimableAccount exists for the given morseSrcAddress:
```

Comment:
ULTRA NIT: using `morseSrcAddress` and `shannon_dest_address`.

Not a big deal. I just have OCD.

Reply:
Good call out - thank you! :pray: